### PR TITLE
[Lightbox media viewer] add isolated image (media) view function

### PR DESCRIPTION
**Pull request by shixiedesign**
### Related Ticket(s)

https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/6534

### Changelog

**Changed** 

- add events to 2.1 Media, where tap and tap again will enter and exist isolated image view

### CODE REVIEW

File: wiki/UI-component/Lightbox-media-viewer.md [hunk 1]
```diff
@@ -85,6 +84,16 @@ Renders an [[Image]] or [[Video]] depending on what the adopters passes.
...
 | `2.1.2` | [[Video]] `ID` | If a [[Video]] `ID` is provided use the [[Video service]] to get [[Video]] data and render |
 
+###### Events
+
+The following events are only applicable to mobile devices.
+
+| event  | description  |
+|:-----------|:-------------|
+| on tap | Triggers isolated media view on mobile. This view allows user to pinch, zoom and drag. |
+| when zoomed out, on tap again | Returns to default view heading and copy present in the viewer |
```
Comment (photodow) [inline wiki/UI-component/Lightbox-media-viewer.md:94]: Missing the reference number on these tables so we can reference them later if needed.

File: wiki/UI-component/Lightbox-media-viewer.md [hunk 2]
```diff
@@ -85,6 +84,16 @@ Renders an [[Image]] or [[Video]] depending on what the adopters passes.
 | `2.1.1` | [[Image]]  | If an [[Image]] is provided, render it as the main media piece.  |
 | `2.1.2` | [[Video]] `ID` | If a [[Video]] `ID` is provided use the [[Video service]] to get [[Video]] data and render |
 
+###### Events
+
+The following events are only applicable to mobile devices.
+
+| event  | description  |
+|:-----------|:-------------|
+| on tap | Triggers isolated media view on mobile. This view allows user to pinch, zoom and drag. |
```
Comment (photodow) [inline wiki/UI-component/Lightbox-media-viewer.md:93]: It's more events than just tap here. Each of the events you called out in this line needs its own row. Like ping, drag to name a couple. We have to be very explicit here, and make sure we get all of them.

**Review by photodow — COMMENTED** [review]
I see this is under media. Is this only for Image though?

I also need to think if there are any other edge cases, or roadblocks we might be neglecting throughout all of this. Have we considered accessibility on this, and how this works with a mouse and keyboard? Or transitioning between a lower breakpoint to a higher breakpoint? Will we need a new component for this? (I'm kind of thinking yes) Are there any transitional animations? Do we need a prototype (if we don't already have one) to make sure we've covered everything?

**Comment by shixiedesign** [comment]
Gonna icebox this for now and wait for other 2 features to merge first.

**Pull request closed without merging**